test/dwd_dynam.jl

```diff
@@ -310,7 +310,76 @@ end
     @test readout(m, vcat(u1, u2, u3), nothing, 0) == [u1 + u2, u2, u3]
     @test eval_dynamics(m, vcat(u1, u2, u3), [x1, x2], nothing, 0) == vcat(x1, u2 + x1 + 2*u2 + u3, x2)
   end
-end
 
+  @testset "VectorInterface for InstantaneousContinuousMachine" begin
+    m = InstantaneousContinuousMachine{Float64,3}(
+      1, 3, 1,
+      (u, x, p, t) -> u*1.5, # dynamics
+      (u, x, p, t) -> u+x[1], # readout
```

Comment:
@slibkind I believe the tests should work logically now, in the `readout` function for this machine I pull out the 3-element vector as the first element of the input vector `x`. And the readout test is now properly `readout(m, u1, [x1], nothing, 0)`.

Reply:
Great!